<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>me</class>
 <widget class="QMainWindow" name="me">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>600</height>
   </size>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>weibo</string>
  </property>
  <property name="styleSheet">
   <string notr="true">border-color: rgb(255, 0, 0);
color: rgb(0, 0, 0);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="userinfo">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>381</width>
      <height>341</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="nickname">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>10</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>昵称</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="introduction">
     <property name="geometry">
      <rect>
       <x>33</x>
       <y>200</y>
       <width>331</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>个人介绍fdfdjkafdkfjkldah;flkadshfkladshfkladshf;dla;ksfhdaslkfhfdkf</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <property name="indent">
      <number>0</number>
     </property>
    </widget>
    <widget class="QCommandLinkButton" name="change_info">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>290</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>个人信息修改</string>
     </property>
    </widget>
    <widget class="QLabel" name="gender">
     <property name="geometry">
      <rect>
       <x>33</x>
       <y>65</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>性    别:</string>
     </property>
    </widget>
    <widget class="QLabel" name="birthday">
     <property name="geometry">
      <rect>
       <x>33</x>
       <y>115</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>生    日</string>
     </property>
    </widget>
    <widget class="QLabel" name="gender_t">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>60</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>女</string>
     </property>
    </widget>
    <widget class="QLabel" name="birthday_t">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>110</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>2018-06-01</string>
     </property>
    </widget>
    <widget class="QCommandLinkButton" name="change_pwd">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>290</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>密码修改</string>
     </property>
    </widget>
    <widget class="QLabel" name="stime_t">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>160</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>2018-06-01</string>
     </property>
    </widget>
    <widget class="QLabel" name="siguptime">
     <property name="geometry">
      <rect>
       <x>33</x>
       <y>165</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>注册时间:</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="history">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>370</y>
      <width>381</width>
      <height>161</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QCommandLinkButton" name="blog">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>29</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>微博</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QCommandLinkButton" name="good">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>100</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>点赞</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="blog_count">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>42</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>1109</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="good_count">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>113</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>200</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="aboutme">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>575</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>我</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="search_">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>575</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>发 现</string>
    </property>
   </widget>
   <widget class="QLabel" name="home">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>575</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>微 博</string>
    </property>
   </widget>
   <widget class="QPushButton" name="message">
    <property name="geometry">
     <rect>
      <x>88</x>
      <y>535</y>
      <width>75</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>imgs/white information.png</normaloff>imgs/white information.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="me_">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>540</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>imgs/black.png</normaloff>imgs/black.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="search">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>540</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>imgs/white search.png</normaloff>imgs/white search.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="home_page">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>530</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>imgs/white house.png</normaloff>imgs/white house.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="msg">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>575</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>消 息</string>
    </property>
   </widget>
   <widget class="QPushButton" name="new_blog">
    <property name="geometry">
     <rect>
      <x>144</x>
      <y>540</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>imgs/plus sign.jpg</normaloff>imgs/plus sign.jpg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="3" margin="6"/>
 <resources/>
 <connections>
  <connection>
   <sender>home_page</sender>
   <signal>clicked()</signal>
   <receiver>home_page</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>47</x>
     <y>545</y>
    </hint>
    <hint type="destinationlabel">
     <x>46</x>
     <y>541</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>message</sender>
   <signal>clicked()</signal>
   <receiver>message</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>130</x>
     <y>546</y>
    </hint>
    <hint type="destinationlabel">
     <x>130</x>
     <y>536</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>new_blog</sender>
   <signal>clicked()</signal>
   <receiver>new_blog</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>201</x>
     <y>550</y>
    </hint>
    <hint type="destinationlabel">
     <x>201</x>
     <y>546</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>search</sender>
   <signal>clicked()</signal>
   <receiver>search</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>272</x>
     <y>549</y>
    </hint>
    <hint type="destinationlabel">
     <x>272</x>
     <y>545</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>me_</sender>
   <signal>clicked()</signal>
   <receiver>me_</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>352</x>
     <y>548</y>
    </hint>
    <hint type="destinationlabel">
     <x>352</x>
     <y>544</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>change_info</sender>
   <signal>clicked()</signal>
   <receiver>change_info</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>120</x>
     <y>303</y>
    </hint>
    <hint type="destinationlabel">
     <x>120</x>
     <y>310</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>blog</sender>
   <signal>clicked()</signal>
   <receiver>blog</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>88</x>
     <y>394</y>
    </hint>
    <hint type="destinationlabel">
     <x>88</x>
     <y>389</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>good</sender>
   <signal>clicked()</signal>
   <receiver>good</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>112</x>
     <y>470</y>
    </hint>
    <hint type="destinationlabel">
     <x>112</x>
     <y>464</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <signal>signal1()</signal>
  <signal>signal2()</signal>
 </slots>
</ui>
